interface/weno/pharmacyHelper.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * pharmacyHelper
+ *
+ *  @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *  This helper is to bridge information from the admin page ajax
+ */
+require_once('../globals.php');
+
+use OpenEMR\Pharmacy\Service\Import;
+
+$city = $_GET['textData'];
+$state = $_GET['state'];
+
+$loadPhar = new Import();
+
+$saved = $loadPhar->importPharmacies($city, $state);
+
+echo $saved;
```

Comment:
Instead of just `echo`'ing the response, let's wrap it in a proper HTTP response. We already have a Symfony component in the codebase to make this super easy.

```php
use Symfony\Component\HTTPFoundation\Respone;

$response = new Response($saved, Response::HTTP_OK, ['content-type' => 'application/json']);
$response->send();
```

Reply:
@robertdown I thank you again. 
Some of this is not working for me. I will go take a Symfony course now and update my javascript skills.  

Usually, the way I build is to get the basic flow working then go back and fancy it up after it is fundamentally working. 
But I see that I am missing a lot of information and need to get up to speed and change/upgrade my coding techniques.

Reply:
There's a real chance I wrote incorrect javascript, I didn't test the code, it was more of an example. Although I'd be happy to pull in your branch and really dig into it if it'd help connect some of the dots.

Reply:
I would be honored to work with you more on this. I went back to free code camp and I see they have added an ES6 section to the JS course. So, it is time to dig in and learn new ways. I did the get the script you wrote to pass the data to the helper but I could not retrieve the objects in the helper. 
This is what I ended up with 

    let f = document.getElementById('addpharmacies');
    f.addEventListener('submit', importPharm);

    function importPharm() {
    top.restoreSession();
    let city = document.getElementById("city").value;
    let state = document.getElementById("state").value;
    if (city.length === 0 || state.length ===0) {
        alert("City and state must both be filled out")  // Do some better error handling here
        return false;
    }
    let body = {"city": city, "state": state};
    let req = new Request('pharmacyHelper.php', {method: "POST", body: body});
    fetch(req)
        .then(response=> {
            if (response.status === 200) {
                return response.json()
            } else {
                throw new Error("Bad response from the server");
            }
        })
        .then(json => {
            alert(json) // Not a great UI experience
        })
   }

In the helper, I added the 

    use Symfony\Component\HttpFoundation\Request;
    use Symfony\Component\HttpFoundation\Response;

When I dumped  $request = Request::createFromGlobals();
I can see two objects. 
I would like your help connecting the dots.

Reply:
Been preoccupied with hurricane stuff the last few days, will get back into this soon